qemu/tests/block_resize.py

```diff
@@ -197,6 +212,10 @@ def get_md5_of_file(filename):
 
         # Verify md5
         if params.get('md5_test') == 'yes':
+            # For linux guest, after guest reboot, if disk letter changed,
+            # reget 'md5_filename'
+            if os_type == 'linux' and disk_letter_changed:
+                md5_filename = mpoint + junction + md5_file
```

Comment:
suggest to move this line to 198, then it does not need to update here(215-218). and disk_letter_changed is removable.

Reply:
@qingwangrh  Updated, thanks.